app/services/campaign_page_builder.rb

```diff
@@ -13,10 +13,9 @@ def initialize(params)
 
   def create_with_plugins
     if page.save
-      create_plugins
+      create_plugins(page)
```

Comment:
`page` is an instance method, but you chosen to pass it as an argument. Is there a reason?